.travis.yml

```diff
@@ -1,11 +1,12 @@
 sudo: false
 language: node_js
-cache:
-  directories:
-    - ~/.npm
+cache: npm
 notifications:
   email: false
-node_js: '8'
+node_js:
+  - '8'
+  - '10'
+  - '12'
```

Comment:
I don't think we need to run this on anything but the latest version of Node because this library is intended to run in the browser.

Reply:
Since the minimum Node version that's supported is 8, I added tests for all versions since 8